src/pages/Components/Toggle/index.md

```diff
@@ -3,35 +3,27 @@ title: 'Toggle'
 order: 7
 ---
 
-> Definition
+> A toggle is used to choose between two possibilities and when the user needs instant feedback. It is common to use toggles when you need to show or hide content and "on/off" switch.
 
 ## Sizes
 
-- m : medium (default)
-- s : small
+You can use two different sizes depending on your needs :
+- m or medium is the default size for toggles on touch screens
+- s or small is mainly used for desktop cases
 
 <preview path="src/pages/Components/Toggle/previews/Toggle" nude="true"></preview>
 
 ## States
 
-- On
-- off
-- on disabled
-- off disabled
+<preview path="src/pages/Components/Toggle/previews/toggle-states" nude="true"></preview>
 
-<preview path="src/pages/Components/Toggle/previews/Toggle" nude="true"></preview>
-
-## do's and don't
-
-when and how should I use and not use the pattern
-
-More infos in [this medium article](https://uxplanet.org/checkbox-vs-toggle-switch-7fc6e83f10b8)
+## Do's and Don't
 
 <hintitem>
-  Always use toggle when option activation 
+  Always use toggle when option activation
 </hintitem>
 <hintitem>
-  Always use toggle when instant activation or feedback 
+  Always use toggle when instant activation or feedback
```

Comment:
yes you did not re-wrote the does and don't, there where reminders notes that needed to be redacted in a more comprehensive way.